INN/umbrella-sfpublicpress
# Interal qa bug fixes

## Changes

This pull request makes the following changes:

- Fixes "more" buttons on articles
![Screen Shot 2020-05-15 at 9 45 04 AM](https://user-images.githubusercontent.com/18353636/82057023-c99a2180-9690-11ea-80bb-a2ea2668d427.png)


- Updates styling for byline date on archives to match dates everywhere else
![Screen Shot 2020-05-15 at 9 46 22 AM](https://user-images.githubusercontent.com/18353636/82057166-f3ebdf00-9690-11ea-8f91-30c1ac68448b.png)
![Screen Shot 2020-05-15 at 9 53 06 AM](https://user-images.githubusercontent.com/18353636/82057800-e3883400-9691-11ea-846c-370a44ebfedd.png)


- Updates header styling on author archive pages to match articles
![Screen Shot 2020-05-15 at 9 47 24 AM](https://user-images.githubusercontent.com/18353636/82057283-1b42ac00-9691-11ea-814d-d59adde67261.png)


- Updates header styling on taxonomy archive pages to match articles
![Screen Shot 2020-05-15 at 9 48 24 AM](https://user-images.githubusercontent.com/18353636/82057369-3c0b0180-9691-11ea-9647-79f77e8e28fd.png)

## Why



For #65

## Testing/Questions

Features that this PR affects:

- Archive headers and byline dates
- Social sharing buttons on articles


Questions that need to be answered before merging:

- [ ] Do we need to update the date format anywhere else?
- [ ] The styling for the author archive headers might change once we decide what to do about GS imported fields vs. default author fields. Details: https://github.com/INN/umbrella-sfpublicpress/issues/65#issuecomment-628194933

Steps to test this PR:

1. View a category or tag archive and make sure all of the dates match the format on single posts.
2. Add a description and featured image to a taxonomy and view its archive. Make sure it displays and looks like a single post header
3. Add info to an author profile and view that authors page. Make sure the header area matches a single post. Also make sure the posts on the author page have the updated date format
